docs/pages/deploy-a-cluster/aws-gslb-proxy-peering-ha-deployment.mdx

```diff
@@ -0,0 +1,306 @@
+---
+title: "AWS Route 53 GSLB Multi-Region Proxy Peering High Availability Deployment Guide"
+description: "Deploying a Proxy-peered High Availability Teleport Cluster using Route 53 to create Global Server Load Balancing"
+---
+
+When deploying Teleport in production, you should design your deployment to
+ensure that users can continue to access infrastructure should an outage or 
+incident affect the availability of your Teleport cluster. 
+
+In order to maintain optimal end-user 
+experience with minimal latency and maximum performance, it is imperative to ensure the scalability 
+of your Auth Service and Proxy Service to accommodate increasing numbers of users and connected resources.
+
+(!docs/pages/includes/cloud/call-to-action.mdx!)
+
+## Overview
+This deployment architecure makes all connected resources accessible through a single Teleport cluster 
+across multiple regions using exclusively AWS ecosystem infrastructure. 
+
+This is accomplished using AWS Route 53 to create Global Server Load Balancing (GSLB) 
+for the Teleport Cluster and Teleport Proxy Peering to reduce the number of connections 
+created through the cluster.
+
+This deployment architecture isn’t recommended for use cases where your users or resources are 
+clustered in a single region or for Managed Service Providers needing to provide separate clusters 
+to customers. Additionally, this architecture is not a solution for increasing the scalability of 
```

Comment:
This sentence is a little confusing.

- First we say "it's imperative to ensure the scalability"
- Then we say "this deployment architecture makes all resources accessible through a single Teleport Cluster"
- Lastly, we say "this is not a solution for increasing the scalability of a single cluster"

Feels a bit like rock paper scissors :-)

Reply:
I think removing "Additionally, this architecture is not a solution for increasing the scalability of 
a single cluster." clears up the confusion. I don't think we need to point out everything this architecture _doesn't do_.